bundles/io.openliberty.tools.eclipse.lsp4e/pom.xml

```diff
@@ -49,15 +59,67 @@
                 <artifactId>maven-dependency-plugin</artifactId>
                     <executions>
                         <execution>
-                            <id>get-mpls</id>
-                            <goals>
-                                <goal>copy-dependencies</goal>
-                            </goals>
-                            <configuration>
-                                <classifier>uber</classifier>
-                                <outputDirectory>${project.basedir}/server/</outputDirectory>  
-                            </configuration>        
-                        </execution>
+				            <id>copy-mpls</id>
+				            <phase>generate-resources</phase>
```

Comment:
Do we have a reason for not using the default phase? https://maven.apache.org/plugins/maven-dependency-plugin/copy-mojo.html